packages/ui/src/Table/Table.tsx

```diff
@@ -422,6 +426,12 @@ export const Table = <D extends object>({
     [onKeyUp],
   )
 
+  useDebouncedEffect(() => {
```

Comment:
Why don't you use this option?
`children(content, tableInstance)`

Reply:
Is `children` supposed to be a function component, which will listen to changes by passed `tableInstance` argument, and will return `content`? If this is the case, the call `children(content, tableInstance)` is not suitable (as it does not have children argument).
If it does not work like that, we can't utilize hooks inside `children` function. We will have to call the hook inside this file (to achieve the same effect), which will prevent us to get rid of the changes in Hive.